src/site/content/en/blog/multi-origin-pwas/index.md

```diff
@@ -0,0 +1,132 @@
+
+## Good and bad uses of multiple origins
+
+There are a few legitimate reasons for sites to employ a multi-origin architecture, mostly related to providing an independent set of web applications, or to create experiences that are completely isolated from each other. There are also uses that should be avoided.
+
+### The good
+
+Let's look at the useful reasons first:
+
+- **Localization / Language:** Using a [country-code top-level domain](https://developer.mozilla.org/en-US/docs/Glossary/TLD), to separate sites to be served in different countries (e.g. `https://www.google.com.ar`), or using subdomains to divide sites targeted to different locations (e.g.: `https://newyork.craigslist.org`) or to offer content for a specific language (e.g. `https://en.wikipedia.org`).
+
+- **Device/Platform:** Using different subdomains, to deliver different versions of a website to different devices. The `m.` or `mobile.` pattern is a common way to separate mobile from desktop in adaptive sites. For example: a site can maintain its desktop version at `https://www.site.com` and mobile version at `https://m.site.com`.
+
+- **Independent webapps:** Using different subdomains to provide experiences whose purpose differs considerably from the site on the main origin. For example, in a news site, the crosswords webapp could be intentionally served from `https://crosswords.site.com`, and installed and used as an independent PWA, without having to share any resources or functionality with the main website.
+
+### The bad
+
+If you're not doing any of these things, it's likely that using a multi-origin architecture will be a disadvantage when building Progressive Web Apps.
+
+Despite this, many sites continue being structured this way for no particular reason, or for 'legacy' reasons. One example is using subdomains to arbitrarily separate parts of a site that should be part of a unified experience.
+
+The following patterns, for example, are highly discouraged:
+
+- **Site sections:** Separating different sections of a site on subdomains. In news sites, it's not uncommon to see the home page at: `https://www.site.com`, while the sports section lives at `https://sports.site.com`, politics at `https://politics.site.com`, and so forth. In the case of an e-commerce site, using `https://www.site.com`, for Home, `https://listings.site.com` for Listings, `https://product.site.com` for Product Pages, for example.
```

Comment:
These really do need to be `example.com`. That site is explicitly reserved by [RFC 2606](https://tools.ietf.org/html/rfc2606). Made-up site names in general could potentially have objectionable or offensive content in the future. site.com in particular redirects to salesforce.com.